### ISSUE TITLE
upload Background Image for a theme

### ISSUE THREAD

**Issue by Vibex** [nebukazar/OneeChan#56]
I'm probably missing something really blatant, but is there no way to use a file directly from my pc for the background image of a custom theme? I know I can just upload it to pomf or something and then link using the url, but I think you should be able to do it directly.

**Comment by nebukazar** [comment]
No, there isn't a way because it was not implemented. You can use base64 though. See [Wiki 1](https://github.com/nebukazar/OneeChan/wiki/Custom-Themes#creating-themes) and [Wiki 2](https://github.com/nebukazar/OneeChan/wiki/Mascots#how-to-convert-my-mascot-to-base64).
